lib/roo_on_rails/railties/http.rb

```diff
@@ -28,10 +28,13 @@ class HTTP < Rails::Railtie
           app.config.middleware.use ::Rack::Deflater
         end
 
-        app.config.middleware.insert_before(
-          ActionDispatch::Cookies,
-          ::Rack::SslEnforcer
-        )
+        # Don't use SslEnforcer in test environment as it breaks Capybara
+        unless Rails.env.test?
+          app.config.middleware.insert_before(
+            ActionDispatch::Cookies,
```

Comment:
Can we avoid referring to `ActionDispatch::Cookies`? `--api` mode doesn't have cookies. In fact, is there some way of removing the middleware in the test env rather than adding it in in non-test envs?